README.md

```diff
@@ -1,4 +1,4 @@
-# Dependency Injection Graph
+# dig - A dependency injection framework for Go
```

Comment:
as long as people don't call it "dee ai gee" i don't care :)
It's always going to be `package dig`, but in official literature probably referred to as DIG.